### DISCUSSION TITLE
Updating Curseforge modpack

### DISCUSSION THREAD

**Discussion by omerhodzic123** [itzg/docker-minecraft-server#1743] [Q&A]
Hey I just wanted some clarification on updating a Curseforge modpack. Do we only have to download the latest zip and replace the old server zip with that one and change the modpack variable in the docker compose to the new zip? Is there anything else we need to do and is player and world data all kept intact?

**Comment by itzg** [comment]
I was going to suggest using [this](https://github.com/itzg/docker-minecraft-server#remove-old-modsplugins) to remove the old modpack mods (leaving the world data, etc); however, I don't think that currently works for the sub directory used for CurseForge modpacks. 

Otherwise, if you have the data directory attached from a host directory, you could stop the container remove the `mods` directory manually, and then start the container with the new modpack name.
